README/ReleaseNotes/v634/index.md

```diff
@@ -129,6 +129,24 @@ The following people have contributed to this new version:
 ## PROOF Libraries
 
 
+## PyROOT
+
+### Deprecation of `TPython::Eval()`
+
+The `TPython::Eval()` method is deprecated and scheduled for removal in ROOT 6.36.
+Its implementation was fragile, and the same functionality can be achieved with `TPython::Exec()`, using a C++ variable that is known to the ROOT interpreter for crossing over from Python to C++. You can use the static `TPython::Result()` value for that.
+
+Example:
+```c++
+// Before, with TPython::Eval()
+std::string stringVal = static_cast<const char*>(TPython::Eval("'done'"));
+std::cout << stringVal << std::endl;
+
+// Now, with TPython::Exec()
+TPython::Exec("ROOT.TPython.Result().stringVal = 'done'");
```

Comment:
That's a good point. Let's wait for feedback from the ATLAS guys who opened this issue whether they think this PR goes in the right direction, and if yes I'll think how to make this more thread safe!

Reply:
Ok, since there was no feedback from ATLAS I went ahead improving the implementation with a lock anyway. Thanks for the suggestion! Now I'll also try `std::variant`.